docs.md

```diff
@@ -0,0 +1,40 @@
+## General Moments
+
+### Everything is a Knowledge Item. 
+#### KnowledgeItem 
+KnowledgeItem - abstract class, that is parental for all other items
+**Unique fields: _title, creation_time,  id: **
+
+
+### Information Classes 
+
+#### TextNote: Inherited from KnowledgeItem
+Storage class for text information. TextNote inherited from KnowledgeItem. 
+**Additional fields: string_text
+
+#### Comment: Inherited from KnowledgeItem
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam ultricies sit amet sapien in posuere. Maecenas accumsan dui ut diam molestie, sed consectetur libero eleifend. Suspendisse potenti
+
+#### Image : Inherited from KnowledgeItem
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam ultricies sit amet sapien in posuere. Maecenas accumsan dui ut diam molestie, sed consectetur libero eleifend. Suspendisse potenti
+
+
+### Storage Classes
+#### Storage : Inherited from KnowledgeItem
+Storage - abstract class to store other items. Inherited from KnoweledgeItem. 
+**Additional fields: container that can store objects of different types, but all of them are inherited from KnowledgeItem 
+
+#### Directory : Inherited from Storage
+Same as storage, but can't contain a Storage objects.
+
+#### TextNoteCollection: Inherited from Storage.
+Can store only TextNote objects.
+
+
+### Other Classes 
+#### LocalStream
```

Comment:
Not sure if the name should be "LocalStream". This is not a stream in C++ terms (although we might want to provide such interface as well), this is some class that manages saving and loading to/from "local filesystem" storage. Let's discuss a better name in a separate issue, or in a chat